packages/core/src/components/ui/Carousel/Carousel.tsx

```diff
@@ -22,7 +22,7 @@ function Carousel({
       testId={testId}
       variant="scroll"
       infiniteMode={false}
-      itemsPerPage={isMobile ? 1 : itemsPerPage}
+      itemsPerPage={isMobile ? 1.6 : itemsPerPage}
```

Comment:
this number (1.6) allows us to see the "preview" of the second item. It means 1 item and the second item preview. 🆒 

https://github.com/vtex/faststore/assets/3356699/bd2c0568-c062-49d5-8a99-a5af6b9eddcf

Reply:
this is not customizable, or is it?

Reply:
you can use override to customize this number, like this [example](https://github.com/vtex-sites/starter.store/pull/242/files#diff-9a0c2db584ab91e7243c5919b2e8aa575c570b4b0380cb64d61bb5d49e8e69e2).


In term of `ProductShelf`, I think we should handle the `itemsPerPage` prop in the `ProductShelf` **section** (we don't have this prop in the `ProductShelf` component (fs/ui). Because we use this same number when setting the skeleton for this section 🤔